src/client.py

```diff
@@ -0,0 +1,47 @@
+"""Implementation of client."""
+
+import socket
+import sys
+
+
+def client(message):
+    """Create a client."""
+    infos = socket.getaddrinfo('127.0.0.1', 5000)
+    stream_info = [i for i in infos if i[1] == socket.SOCK_STREAM][0]
+    client = socket.socket(*stream_info[:3])
+    eight_mult = False
+    try:
+        client.connect(stream_info[-1])
+        if len(message) % 8 == 0:
```

Comment:
Better to use a global constant for your buffer length, so it will be guaranteed to be consistent throughout your program.